CognitiveVRTest/Plugins/CognitiveVR/Source/CognitiveVR/Public/DynamicObject.h

```diff
@@ -43,7 +43,7 @@ class COGNITIVEVR_API UDynamicObject : public USceneComponent
 	float currentTime = 0;
 	TSharedPtr<FDynamicObjectId> ObjectID;
 	FVector LastPosition;
-	FVector LastForward;
+	FQuat LastRotation;
```

Comment:
```suggestion
	// previously the component used a dot product from the current forward vector to the last forward vector to determine if the object has rotated
	// If there is a rotation only along the forward vector, the rotation will not record this change.
	// Changing to use Quaternions instead to fix this issue
	FQuat LastRotation;
```

Suggesting a comment to explain the change from vector to quaternion.